contracts/babylon/src/ibc.rs

```diff
@@ -110,9 +110,9 @@ pub fn ibc_channel_close(
     // TODO: erase all contract state upon closing the channel
 }
 
-/// Invoked when an IBC packet is received
-/// We decode the contents of the packet and if it matches one of the packets we support
-/// execute the relevant function, otherwise return an error.
+/// Invoked when an IBC packet is received.
+/// We decode the contents of the packet and if it matches one of the packets we support, execute
+/// the relevant function, otherwise return an error.
 /// NOTE: In its current form, this method does not modify state.
```

Comment:
@SebastianElvis , is still the case here? What does 'modify state' means btw? Modify state in case of error?

Reply:
Hmm, not the case anymore as this function indeed modifies state

Reply:
Will update the comments, as this is confusing / misleading.